src/main/java/uk/nhs/prm/repo/ehrtransferservice/database/TransferStore.java

```diff
@@ -33,8 +33,8 @@ public void createEhrTransfer(RepoIncomingEvent incomingEvent, String status) {
                             status,
                             getTimeNow(),
                             getTimeNow(),
-                            getLargeEhrCoreMessageId(),
-                            isActive()
+                            "", //This is a placeholder for the messageId from the LargeEhrCoreQueue,
```

Comment:
Would it be better to instead initialising this to an empty string here, create an additional constructor within `Transfer.java` that just initialises it there and does not require it?

Reply:
This is something I'd like to address properly later down the line, there is a very large number of constructor calls for Transfer and the vast majority use 'true' for isActive. For now I'll leave it as-is, but I agree, I don't like this approach either.

![image](https://github.com/nhsconnect/prm-repo-ehr-transfer-service/assets/127403642/1f9f1703-1a1c-4d55-9002-dc77dc99f62c)